### ISSUE TITLE
"dnstap-output" distorts rendering of subsequent options

### ISSUE THREAD

**Issue by kempniu** [egberts/vim-syntax-bind-named#5]
I am observing a rather funky highlighting effect with `dnstap-output`:

![dnstap-output-dnstap-all](https://user-images.githubusercontent.com/1301146/80343606-7c334d00-8866-11ea-9ee4-0c45af7d7dc8.png)

Without `dnstap { all; };`:

![dnstap-output](https://user-images.githubusercontent.com/1301146/80343403-18a91f80-8866-11ea-9559-08621a498995.png)

And here is `dnstap { all; };` without `dnstap-output` (looks as expected):

![dnstap-all](https://user-images.githubusercontent.com/1301146/80343476-3c6c6580-8866-11ea-9312-4e11853cde21.png)

Here is the sample configuration file for easy copy-paste:

```
options {
	dnstap-output file "dnstap.out";
	dnstap { all; };
	send-cookie no;
	require-server-cookie no;
	minimal-responses no;
	dnssec-validation yes;
	qname-minimization disabled;
};
```

I would be happy to help with debugging this and/or test any potential fixes.

**Comment by egberts** [comment]
This "funky" highlighting serves as a backwood trail to the offending line; just that the offending line isn't offensive.

Am on it.

**Comment by egberts** [comment]
FOREWORD:  any dark-cyan highlighting is that error spot.  (It's supposed to be red, but that my next task).

Progress:  'dnstap' keyword is found in both  'view' and 'options'.  The `namedOV_Dnstap` is that starting (or ending) point.  It has the necessary highlighting and everything but this rule didn't fire within Vim 8.1.

So  the next step is to look  at the previous statement: `dnstap-output ...`.  Taking that out will make the `dnstap { all; };` and the rest of the options work and properly highlighted again.

So, the cultprit is the `dnstap-output` rule.   And as a detector, the ending semicolon symbol to the `dnsstap-output` rule was highlighted as dark-cyan (it should have been soft green).

On a separate slightly related note: this same 'dnstap { all; };' doesn't work inside a 'view' clause. :-/

**Comment by egberts** [comment]
Fixed it in 9394ac15f346236e612944f332c747f81401d449

**Comment by egberts** [comment]
Thank you for the issue report, @kempniu
